.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  plugins: [],
+  rules: {
+    indent: ["error", 2],
+    semi: ["error", "always"],
+    "no-redeclare": "off",
+    "no-console": "off"
+  },
+  env: {
+    browser: true,
+    mocha: true,
+    jquery: true,
+    es6: true
+  },
+  globals: {},
+  "parserOptions": {
+    "sourceType": "module",
```

Comment:
I think this config is not needed at all, since we don;t have module in this repo, just a script.

Reply:
If use "import" or "export" need to use this property, because if not use this, ESLint acsuse the error "Parsing error: 'import' and 'export' may appear only with 'sourceType: module'"